packages/core/src/external/cda/remove-b64.ts

```diff
@@ -22,17 +23,21 @@ export function removeBase64PdfEntries(payloadRaw: string): string {
       if (
         toArray(comp.section?.templateId).some(
           //eslint-disable-next-line @typescript-eslint/no-explicit-any
-          (template: any) => template?.["@_root"] === notesTemplateId
+          (template: any) =>
+            template?.["@_root"] === notesTemplateId || template?.["@_root"] === resultsTemplateId
         )
       ) {
         if (comp.section.entry) {
           //eslint-disable-next-line @typescript-eslint/no-explicit-any
           comp.section.entry = toArray(comp.section.entry).filter((entry: any) => {
             const mediaType = entry.act?.text?.["@_mediaType"]?.trim().toLowerCase();
             if (
-              BINARY_MIME_TYPES.includes(mediaType) &&
-              entry.act.text["@_representation"]?.trim().toLowerCase() ===
-                b64Representation.toLowerCase()
+              (BINARY_MIME_TYPES.includes(mediaType) || mediaType === undefined) &&
```

Comment:
nit: I'd use `==` to check for nullish/emptyness so it considers `null` as well.